src/fontra/views/fontinfo/panel-sources.js

```diff
@@ -428,13 +455,62 @@ class SourceBox extends HTMLElement {
     }
   }
 
-  toggleShowHide() {
-    const element = this.querySelector("#open-close-icon");
-    element.classList.toggle("item-closed");
+  toggleShowHide(altKey) {
+    const cardElements = !altKey
+      ? [this]
+      : document.querySelectorAll(".fontra-ui-font-info-sources-panel-source-box");
+
+    const thisIconElement = this.querySelector("#open-close-icon");
+    const isClosed = thisIconElement.classList.contains("item-closed");
+
+    for (const cardElement of cardElements) {
+      const elementIcon = cardElement.querySelector("#open-close-icon");
+      if (isClosed) {
+        cardsInfos[cardElement.sourceIdentifier].isClosed = false;
+        cardElement.classList.remove("item-closed");
+        elementIcon.classList.remove("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.remove("min-height");
+        }
+      } else {
+        cardsInfos[cardElement.sourceIdentifier].isClosed = true;
+        cardElement.classList.add("item-closed");
+        elementIcon.classList.add("item-closed");
+        for (const child of cardElement.children) {
+          child.classList.add("min-height");
+        }
+      }
+    }
+  }
+
+  getOneliner() {
+    const onelinerElement = html.div({
+      class: "fontra-ui-font-info-sources-panel-oneliner",
+    });
+    onelinerElement.appendChild(
+      html.span({ class: "source-name" }, [this.source.name])
+    );
+
+    for (const [i, axis] of enumerate(this.fontAxesSourceSpace)) {
+      if (i > 0) {
+        onelinerElement.append(", ");
+      }
+      const axisElement = document.createElement("span");
+      const sourceLocationValue = round(
+        this.source.location.hasOwnProperty(axis.name)
+          ? this.source.location[axis.name]
+          : axis.defaultValue,
+        2
+      );
+      axisElement.innerText = `${axis.name}=${sourceLocationValue}`;
 
-    for (const child of this.children) {
-      child.classList.toggle("min-height");
+      if (axis.defaultValue != sourceLocationValue) {
```

Comment:
TBH I'd flip this logic around: it's the default value that needs to be styled differently. I'd use a class named "axis-is-at-default", and add the gray styling there.

Reply:
I had this before, but with the new approach of handling the ', ' -> the commas were black by default. That's why I flipped it.

Reply:
done.